prac_05/word_occurrences.py

```diff
@@ -0,0 +1,30 @@
+def main():
+    user_input = input("Text: ")
+    word_counts = count_word_occurrences(user_input)
+    display_word_occurrences(word_counts)
+
+
+def count_word_occurrences(text):
+    word_counts = {}
+
+    words = text.split()
+    for word in words:
+        word = word.lower()
+        if word in word_counts:
+            word_counts[word] += 1
+        else:
+            word_counts[word] = 1
+
+    return word_counts
+
+
+def display_word_occurrences(word_counts):
+    display_order = ['this is a collection of words of nice words this is a fun thing it is']
```

Comment:
It initializes a list display_order, which currently contains a single string representing the expected order of words.